### PR TITLE
Portofolio Section

### PR DESCRIPTION
**Pull request by iamdebajyotibanerjee**
Instead of 3 column grid, I've created a bigger 1 column grid for better visual effect. Also I've givven a hover effect on the portfolio sites.

### CODE REVIEW

**Comment by empeje** [comment]
Assuming it has the structure code created by @devkonyan it related to #4 and #7

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+/node_modules
```
Comment (empeje) [inline .gitignore:2]: It should has a empty line ending

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <section class="carrosel">
+
+    </section>
+    <section class="about">
+
+      </section>
+      <section class="container-fluid portfolio">
+          <div class="row">
+           <div class="col-md-8 col-md-offset-2 text-center portfolio-1">
+               <a href="https://www.donut.ai">
```
Comment (empeje) [inline index.html:34]: whose portfolio it is?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                   <img src="img/Donut.png" alt="Portfolio 1" class="img-responsive">
+                 </a>                
+                  <p>Portfolio 1</p>
```
Comment (empeje) [inline index.html:37]: whose portfolio it is? I think we should link to our team project.

Perhaps we can create other projects after this landing page 🤔

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                  <p>Portfolio 4</p>
+              </div>
+          </div>
+    </section>
+    <footer class="footer">
+
+    </footer>
+</body>
+
+</html>
```
Comment (empeje) [inline index.html]: new line ending please

**Review by empeje — CHANGES_REQUESTED** [review]
Good but, need add how to use it in `README.md`

File: index.html (hunk 2 above)
Comment (iamdebajyotibanerjee) [reply index.html:34]: It's a portfolio from Chingu showcase.

File: index.html (hunk 3 above)
Comment (iamdebajyotibanerjee) [reply index.html:37]: We can discuss it

**Comment by debu2code** [comment]
Done

File: index.html (hunk 4 above)
Comment (iamdebajyotibanerjee) [reply index.html]: Done

File: .gitignore (hunk 1 above)
Comment (iamdebajyotibanerjee) [reply .gitignore:2]: Done

**Review by empeje — APPROVED** [review]
🆗 👌

**Pull request merged**